manager/integration/tests/test_upgrade.py

```diff
@@ -490,6 +614,24 @@ def test_upgrade(longhorn_install_method,
         assert sspod.status.container_statuses[0].restart_count == 0, \
             f"assert pod {sspod_info['pod_name']} didn't restart after upgrade"
 
+    # Attach all detached volume after system upgrade
+    vol_detached = client.by_id_volume(vol_detached_name)
+    vol_detached.attach(hostId=host_id)
+    vol_detached = wait_for_volume_healthy(client, vol_detached_name)
+
+    vol_detached_with_data = client.by_id_volume(vol_detached_with_data_name)
+    vol_detached_with_data.attach(hostId=host_id)
+    vol_detached_with_data = \
+        wait_for_volume_healthy(client, vol_detached_with_data_name)
+
+    vol_detached_v2 = client.by_id_volume(vol_detached_v2_name)
+    vol_detached_v2.attach(hostId=host_id)
+    vol_detached_v2 = wait_for_volume_healthy(client, vol_detached_v2_name)
```

Comment:
Due to these three issues:
[Issue #10461](https://github.com/longhorn/longhorn/issues/10461)
[Issue #10131](https://github.com/longhorn/longhorn/issues/10131)
[Issue #10053](https://github.com/longhorn/longhorn/issues/10053)
I believe `v2 volume attach test` can only be applied to versions after `1.9.x` and is not suitable for versions prior to `1.8.x`.

Reply:
Thank you, I have implemented this part in local, will update after my local test complete.